README.md

```diff
@@ -46,6 +46,9 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [XACC](https://github.com/ORNL-QCI/xacc) - Extreme-scale programming model for quantum acceleration within high-performance computing ([arXiv paper](https://arxiv.org/abs/1710.01794)).
 - [qpp](https://github.com/softwareQinc/qpp) - Quantum++ is a modern C++ general purpose quantum computing library, composed solely of template header files.
 
+**Assembly**
+- [Quplexity](https://github.com/MrGilli/Quplexity) - Modular library/toolkit for QC sims, including key mathematical functions and logic its written in ARM(64) and x86 Assembly, which results in improved performance and "weight" benefits.
```

Comment:
> logic its written

Is the "its" a typo?

Reply:
Sorry my apologies, I will fix It and create another pull request.

Reply:
Ok, the mistake is fixed in the new pull request, sorry.